### ISSUE TITLE
Resizing columns cause column headers to become malformed

### ISSUE THREAD

**Issue by joshparolin** [adazzle/react-data-grid#510]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
When resizing a column, the column headers become malformed thus becoming misaligned with columns below them.

**Expected/desired behavior**
When resizing a column, the rest of the column headers adjust to the new width of the column.

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem. I believe this has something to do with the latest perf updates.

![gif](http://g.recordit.co/YtgnkfEDE5.gif)

**Comment by dkwin** [comment]
@joshparolin , I had this problem before because webpack was loading two versions of React when I was building the bundle.

In my project, I had Reactv15 but react grid requires React v14, so webpack loaded both versions.  Maybe you can check if you have 2 versions of React loaded.

**Comment by therahl** [comment]
@dkwin I can confirm that we do not have two versions of react in our project, but still experiencing this issue.  The gif from @joshparolin was taken from the react-data-grid website, where I would assume they do not have two versions of React loaded since they do not support React 15.  

A side note, we tried pulling in the latest react-data-grid into our fork, hoping to take advantage of the virtualization improvements, and there were several UI issues including the column sizing.  We ultimately decided to rollback and use an earlier version, until things aren't as broken.

**Comment by supamanda** [comment]
Looks like this is fixed now, with #515
